Hardware/CMockHardwareMap.cpp

```diff
@@ -335,25 +333,25 @@ CMockHardwareMap::command_error_code_t CMockHardwareMap::setRating(
     {
         m_heater_rating[channel - 1] = rating;
     }
-    return COMMAND_OK;
+    return ICommand::COMMAND_OK;
 }
 
-CMockHardwareMap::command_error_code_t CMockHardwareMap::setCapacity(
+ICommand::command_error_code_t CMockHardwareMap::setCapacity(
     ICommand *p_command)
 {
     // Sanitise command arguments
     if ((p_command->getArgumentCount() < 2) ||
         (p_command->getArgumentCount() > 3))
     {
-        return ERROR_ARG_COUNT;
+        return ICommand::ERROR_ARG_COUNT;
     }
     float heater_capacity = (*p_command)[0];
```

Comment:
Magic numbers make code harder to read and understand. Arguments can be made to use enums for better readability and safety.

Reply:
Make a suggestion how to use enums.